Participate/web-content-call.md

```diff
@@ -0,0 +1,5 @@
+### Web Content
+
+This working group maintains the website and helps coordinate the distribution of CHAOSS messaging on internet platforms. Areas of interest include the website, Twitter, CHAOSScast, CHAOSStube, CHAOSSweekly, CHAOSSblog, CHAOSS metrics release, and press releases.
+
+The Web Content community call meets the First Monday of each month at 11:00am CT (usually 17:00 (CET), [check your local time]()) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes]()
```

Comment:
I don't know how to create a "are we meeting yet" link for a "first Monday of the month" meeting.
I suggest removing it.

Still missing a GoogleDoc URL.

```suggestion
The Web Content community call meets the First Monday of each month at 11:00am CT (usually 17:00 CET) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes]()
```

Reply:
Okay, I have added the link and removed the check